python/README.md

```diff
@@ -30,13 +30,15 @@ The first API call exposed is
 It can provide summary information on a single (via a GET) or multiple
 (via a POST) url or urls.
 
-	# Let's get some URL metrics. Results are now an array of dictionaries
-	# the i'th dictionary is the results for the i'th URL
-	metrics = l.urlMetrics(['www.moz.com', 'www.moz.com'])
+    # Let's get some URL metrics. Results are now an array of
+	# dictionaries the i'th dictionary is the results for the i'th URL
```

Comment:
This funky indentation is a result of tabs in the existing file. I'll clean them up in a separate PR after this one, but I'll leave them alone in this PR to keep the diff clear.